pkg/cephfs/driver.go

```diff
@@ -127,6 +127,10 @@ func (fs *Driver) Run(driverName, nodeID, endpoint, volumeMounter, mountCacheDir
 		csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER,
 	})
 
+	fs.cd.AddControllerServiceCapabilities([]csi.ControllerServiceCapability_RPC_Type{
```

Comment:
this is also not needed  as we are not supporting expand volume

Reply:
Done